content/en/docs/Writing policies/jmespath.md

```diff
@@ -0,0 +1,309 @@
+            "version": "v1",
+            "resource": "pods"
+        },
+        "requestKind": {
+            "group": "",
+            "version": "v1",
+            "kind": "Pod"
+        },
+        "requestResource": {
+            "group": "",
+            "version": "v1",
+            "resource": "pods"
+        },
+        "name": "mypod",
+        "namespace": "foo",
+        "operation": "CREATE",
+        "userInfo": {
+            "username": "thomas",
+            "uid": "404d34c4-47ff-4d40-b25b-4ec4197cdf63"
+        },
+        "object": {
+            "kind": "Pod",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "mypod",
+                "creationTimestamp": null
+            },
+            "spec": {
+                "containers": [
+                    {
+                        "name": "busybox",
+                        "image": "busybox",
+                        "resources": {}
+                    }
+                ]
+            },
+            "status": {}
+        },
+        "oldObject": null,
+        "dryRun": false,
+        "options": {
+            "kind": "CreateOptions",
+            "apiVersion": "meta.k8s.io/v1"
+        }
+    }
+}
+```
+
+As can be seen, the full Pod is represented along with other metadata surrounding its creation.
+
+These AdmissionReview resources serve as the most common source of data when building JMESPath expressions, specifically `request.object`. For the other data properties which can be consumed via an AdmissionReview resource, refer back to the [variables page](/docs/writing-policies/variables/#variables-from-admission-review-requests).
+
+## Formatting
+
+Because there are various types of values in differing fields, there are differing ways values must be supplied to JMESPath expressions as inputs in order to generate not only a valid expression but produce the output desired. Specifying values in the correct format is key to this success. Values which are supported but need to be differentiated in formatting are numbers (i.e., an integer like `6` or a floating point like `6.7`), a quantity (i.e., a number with a unit of measure like `6Mi`), a duration (i.e., a number with a unit of time like `6h`), a [semver](https://semver.org/) (i.e., a version number like `1.2.3`), and others. Because Kyverno (and therefore most custom JMESPath filters built for Kyverno) is designed for Kubernetes, it is Kubernetes aware. Therefore, specifying \`6\` as an input to a filter is not the same as specifying \'6\' where the former is interpreted as "the number six" and latter as "six bytes". The _types_ which map to the possible values are either JSON or string. In JMESPath, these are [literal expression](https://jmespath.org/specification.html#literal-expressions) and [raw string literals](https://jmespath.org/specification.html#raw-string-literals). Use the table below to find how to format the type of value which should be supplied.
```

Comment:
Quantities should be surrounded in quotes instead of back ticks.

Reply:
Sorry, not following. Where is the error here?

Reply:
Ah sorry, my bad. Got confused between the markdown usage of backticks and the jmespath usage of backticks. I thought the documentation said to use backticks to represent durations and quantities in the jmespath expression.

Reply:
No problem. Marking resolved.